app/src/main/java/com/onboarding/parkingsystemkotlin/mvp/presenter/ConfigureParkingPresenter.kt

```diff
@@ -0,0 +1,25 @@
+package com.onboarding.parkingsystemkotlin.mvp.presenter
+
+import com.onboarding.parkingsystemkotlin.listener.OnInputListener
+import com.onboarding.parkingsystemkotlin.mvp.contract.ConfigureParkingLotsContract
+
+class ConfigureParkingPresenter(
+    private val view: ConfigureParkingLotsContract.ConfigureParkingView,
+    private val inputListener: OnInputListener
+) : ConfigureParkingLotsContract.ConfigureParkingPresenter {
+    override fun onOkButtonPress() {
+        val lots: String = view.getLots()
+        if (lots.isNotEmpty()) {
+            val lotsInt = lots.toInt()
+            inputListener.sendInput(lotsInt)
```

Comment:
why are you using this listener here? it should be in the view

Reply:
done